CONTRIBUTING.rst

```diff
@@ -448,6 +448,55 @@ fetch and checkout a remote branch like this::
     https://notes.envato.com/developers/rebasing-merge-commits-in-git/
 .. _`Rebase`: https://help.github.com/rebase/
 
+.. _contributing-docker-development:
+
+Developing and Testing with Docker
+----------------------------------
+
+Because of the many components of celery, such as a broker and backend,
```

Comment:
Some really minor points; better use `Docker` instead of `docker` everywhere and `Celery` instead of `celery`.